svgelements/svgelements.py

```diff
@@ -5497,7 +5500,7 @@ def _validate_subpath(self, index):
             if isinstance(close_search, Close):
                 for k in range(index, -1, -1):
                     move_search = self._segments[k]
-                    if isinstance(move_search, Move):
+                    if isinstance(move_search, (Move, Close)):
                         self._segments[j].end = Point(move_search.end)
```

Comment:
This is clearly searching for the last Move. And should not search for the last Close. While these should turn out the be the same point, and thus a moot point. It should be previous Move only.